src/otx/config/callbacks/default.yaml

```diff
@@ -1,22 +1,47 @@
-defaults:
-  - model_checkpoint
-  - early_stopping
-  - model_summary
-  - rich_progress_bar
-  - _self_
+# https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.callbacks.EarlyStopping.html
+early_stopping:
+  _target_: lightning.pytorch.callbacks.EarlyStopping
+  monitor: ??? # quantity to be monitored, must be specified !!!
+  min_delta: 0. # minimum change in the monitored quantity to qualify as an improvement
+  patience: 3 # number of checks with no improvement after which training will be stopped
+  verbose: False # verbosity mode
+  mode: "min" # "max" means higher metric value is better, can be also "min"
```

Comment:
How about changing this default value to "max" ? I think most of our case, we'll use the "max" Acc, IoU, mAP, etc.

Reply:
We'll discuss this further, but for now, default alone doesn't currently work without overriding.